sub-pages/_vars-sub-pages.html

```diff
@@ -15,4 +15,9 @@
 {# This to fake data for records sub-pages #}
 {# TODO: Update _queries/records.json when we have real record posts #}
 {% set record_query = queries.records %}
-{% set records = record_query.search_with_url_arguments(size=10) %}
\ No newline at end of file
+{% set records = record_query.search_with_url_arguments(size=10) %}
+
+{# Forms for sub-pages #}
+{% set forms = ['Amending-and-correcting-records-under-the-Privacy-Act-form',
```

Comment:
This is sorta up in the air because of this whole tech change conversation, but as it stands now it would be prudent to put these "form-slugs" into Wordpress. That being said, I think this is something we should talk about as a group to find that right way to do it.